doc/source/tutorial/stats/plots/qmc_plot_curse.py

```diff
@@ -0,0 +1,38 @@
+"""Visualize the curse-of-dimensionality.
+
+It presents a saturated design in 1, 2 and 3 dimensions for a
+given discretization.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+disc = 10
+
+x = np.linspace(0, 1, disc)
+y = np.linspace(0, 1, disc)
+z = np.linspace(0, 1, disc)
+
+xx, yy, zz = np.meshgrid(x, y, z)
+
+fig = plt.figure(figsize=(12, 4))
+ax = fig.add_subplot(131)
+ax.set_aspect('equal')
+ax.scatter(xx, yy * 0)
+ax.set_xlabel(r'$x_1$')
+ax.get_yaxis().set_visible(False)
+
+ax = fig.add_subplot(132)
+ax.set_aspect('equal')
+ax.scatter(xx, yy)
+ax.set_xlabel(r'$x_1$')
+ax.set_ylabel(r'$x_2$')
+
+ax = fig.add_subplot(133, projection='3d')
+ax.set_box_aspect((1, 1, 1))
```

Comment:
This doesn't work in my installed version of Matplotlib (3.3.2). And changing to `set_aspect` gives:
```
NotImplementedError: It is not currently possible to manually set the aspect on 3D axes
```
Probably related to https://github.com/matplotlib/matplotlib/pull/17515, which fixed up the default behavior. Given that we use latest released `matplotlib` to render the docs, I think it's best to delete this line (pushing a fix).

Reply:
Yes I had trouble with this too. I had an older version with another way to set this. Seems like they keep changing the behavior here...